src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -24,9 +24,9 @@ <h2 class="addon"{{ addon.name|locale_html }} id="addon-name">
       <span class="version">{{ version.version }} ({{ version.get_channel_display() }})</span>
     {% endif %}
   </h2>
-  <h4 class="author">by
-    {% for author in addon.listed_authors %}
-    <a href="{{ url('reviewers.developer_profile', author.id) }}">{{ author.name }}</a>{% if not loop.last %},{% endif %}
+  <h4 class="author">by 
+    {% for author in addon.current_authors %}
+    <a href="{{ url('reviewers.developer_profile', author.id) }}"{% if author not in addon.listed_authors %}class="is_unlisted"{% endif %}>{{ author.name }}</a>{% if not loop.last %},{% endif %}
```

Comment:
The `Addon.attach_authors` adds a `role` and `listed` properties to the `UserProfile` instances it gathers, so you should be able to do this in a slightly more efficient way:

```suggestion
    <a href="{{ url('reviewers.developer_profile', author.id) }}"{% if not author.listed %}class="is_unlisted"{% endif %}>{{ author.name }}</a>{% if not loop.last %},{% endif %}
```